feature-group-definitions/media-capture.yml

```diff
@@ -1,5 +1,11 @@
 spec: https://w3c.github.io/html-media-capture/
 caniuse: html-media-capture
+status:
+  baseline: false
```

Comment:
Let's split this into a separate PR to figure out https://github.com/web-platform-dx/web-features/pull/461#issuecomment-1839195349

Reply:
Moved to https://github.com/web-platform-dx/web-features/pull/500